src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs

```diff
@@ -218,9 +218,9 @@ internal static int GetHashCode(ICollection<NameValueHeaderValue> values)
             }
 
             var result = 0;
```

Comment:
This code already has like 5 other hash functions. I don't want to change one without changing the others, and I don't want to puzzle out all of the knots in the others with no real clear reason to change them. This was ported from an exiting vOld codebase.